lib/safe-pg-migrations/helpers/satisfied_helper.rb

```diff
@@ -3,10 +3,24 @@
 module SafePgMigrations
   module Helpers
     module SatisfiedHelper
-      private
+      class << self
+        def satisfies_change_column_null_requirements?
+          satisfies_add_check_constraints? && SafePgMigrations.pg_version_num >= 120_000
```

Comment:
https://github.com/ankane/strong_migrations#setting-not-null-on-an-existing-column speaks about a difference in PG 12 that allows us to alter table after adding the constraint fearlessly. I couldn't find a reference to that somewhere else, so I decided to trust strong_migration